langgraph/src/pregel/read.ts

```diff
@@ -143,21 +184,33 @@ export class ChannelInvoke<
 
   pipe<NewRunOutput>(
     coerceable: RunnableLike
-  ): ChannelInvoke<RunInput, Exclude<NewRunOutput, Error>> {
-    if (this.bound === defaultRunnableBound) {
-      return new ChannelInvoke<RunInput, Exclude<NewRunOutput, Error>>({
+  ): PregelNode<RunInput, Exclude<NewRunOutput, Error>> {
+    if (ChannelWrite.isWriter(coerceable)) {
+      return new PregelNode<RunInput, Exclude<NewRunOutput, Error>>({
+        channels: this.channels,
+        triggers: this.triggers,
+        mapper: this.mapper,
+        writers: [...this.writers, coerceable as ChannelWrite],
+        bound: this.bound.pipe(coerceable),
```

Comment:
This diverges from the Python implementation. I couldn't get the existing tests to pass without piping `coerceable`. Any advice?

Python implementation: https://github.com/langchain-ai/langgraph/blob/main/langgraph/pregel/read.py#L201

Reply:
That's because you need other changes in Pregel class I think

Reply:
Is it ok to leave this change in for now?

Reply:
yes, but make a note to change this